modules/view-compliance-status-for-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-compliance.adoc
+:_module-type: PROCEDURE
+[id="view-compliance-status-for-cluster_{context}"]
+= View compliance status for clusters
+
+[role="_abstract"]
+You can view compliance state for all clusters or a single cluster from the compliance dashboard.
+
+.Procedure
+* To view compliance status for all clusters in your environment:
+
+. Navigate to the RHACS portal and open the compliance dashboard by selecting *Compliance* from the left-hand navigation menu.
```

Comment:
Since it's technically a level 2 under a bullet, I think you'd want to do `..` for each of these numbered steps. Here and in other modules.

Reply:
The preview looks fine though 🤔 

![image](https://user-images.githubusercontent.com/23069445/123352089-ee560d80-d5a1-11eb-85c5-4f8bc8e05f72.png)

VS

![image](https://user-images.githubusercontent.com/23069445/123352218-3117e580-d5a2-11eb-81f8-9f70c7762177.png)

Should I change it still?

Reply:
Yes, we'd want it showing at the proper level. If you just use the one `.`, it'll show as a number. But if you use the two `..`, it shows as an a./b./c. , etc. 

It looks like you changed it for the first part ("To view compliance status for all clusters..."), which shows a/b properly, but not for the second part ("To view compliance status for a specific cluster ..."), which shows 1/2/3.

So if you can fix this second part, this PR is ready to merge!